From fc7914b4a239c436e39e7ea89e7005361acd63bb Mon Sep 17 00:00:00 2001
From: Michael Sarahan <msarahan@gmail.com>
Date: Wed, 19 Dec 2018 11:18:53 -0600
Subject: [PATCH] fix library names for openssl 1.1.1 on win

---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 4446ee5..2db8bef 100644
--- a/setup.py
+++ b/setup.py
@@ -403,8 +403,8 @@ For further information please check the 'doc/src/install.rst' file (also at
                     self.library_dirs.append(os.path.join(path, "ms"))
                     break
             if self.have_ssl:
-                self.libraries.append("libeay32")
-                self.libraries.append("ssleay32")
+                self.libraries.append("libssl")
+                self.libraries.append("libcrypto")
                 self.libraries.append("crypt32")
                 self.libraries.append("user32")
                 self.libraries.append("gdi32")
-- 
2.19.1

